portfolio/src/main/webapp/index.html

```diff
@@ -61,5 +61,34 @@ <h2>Hello, my name is Christian Liguori.</h2>
         </div>
       </div>
     </div>
+
+    <!-- Contact Form -->
+    <div id="contact-form-container">
+      <div class="container">
+        <div class="row">
+          <div class="col s10 offset-s1">
+            <h3>Get in touch!</h3>
+            <form 
+              action="/form-handler" 
+              method="POST">
+              <input 
+                type="text"
+                name="sender-name">
+              <label for="sender-name">What's your name?</label>
+              <input 
+                type="text"
+                name="sender-organization">
+              <label for="sender-organization">Where do you work?</label>
+              <input 
+                type="text"
+                name="sender-message">
+              <label for="sender-message">What's on your mind?</label>
+              <br>
```

Comment:
nit: Use CSS to change the visual layout and positioning of elements on the page rather than `<br>` here.

Reply:
Okay, I will use css to handle the positioning.